### DISCUSSION TITLE
Request / Response body schemas in the API docs frontend

### DISCUSSION THREAD

**Discussion by marhkb** [containers/podman#21806] [General]
Has anyone ever noticed that https://docs.podman.io/en/latest/_static/api.html does't provide useful schemas for request/response bodies? Am I missing something?

Some examples

## Response Body Schema

[Docker (list images)](https://docs.docker.com/engine/api/v1.44/#tag/Image/operation/ImageList)
![grafik](https://github.com/containers/podman/assets/3630213/6f2a07e7-e894-4fbb-9eef-ed1b30fa66e6)

[Podman (list images)](https://docs.podman.io/en/latest/_static/api.html#tag/images/operation/ImageListLibpod):
![grafik](https://github.com/containers/podman/assets/3630213/ebaf320e-889c-412d-bfff-a855d7167dbd)

## Request Body Schema

[Docker (create volume)](https://docs.docker.com/engine/api/v1.44/#tag/Volume/operation/VolumeCreate):
![grafik](https://github.com/containers/podman/assets/3630213/94bbddca-d1d2-42a3-9ab6-449c3bb40044)

[Podman (create volume)](https://docs.podman.io/en/latest/_static/api.html#tag/volumes/operation/VolumeCreateLibpod)
![grafik](https://github.com/containers/podman/assets/3630213/81a7943c-ccc2-4f21-bfa2-9a82846b4348)

**Comment by baude** [comment]
this is all based on swagger and we are not super diligent about it...you can certainly file issues, or better yet, give us a hand with a contribution pull request!
